### ISSUE TITLE
Error in go.nvim

### ISSUE THREAD

**Issue by voddle** [ray-x/go.nvim#174]
When I add
`require('go').setup()`
to init.vim and source it, error below orrur, I am wondering what can I do to fix it
```
E5108: Error executing lua /private/var/root/.vim/plugged/go.nvim/lua/go/commands.lua:161: attempt to call upvalue 'create_cmd' (a nil value)
stack traceback:
        /private/var/root/.vim/plugged/go.nvim/lua/go/commands.lua:161: in function 'add_cmds'
        /private/var/root/.vim/plugged/go.nvim/lua/go.lua:124: in function 'setup'
        [string ":lua"]:1: in main chunk

```

**Comment by voddle** [comment]
Sry for this issue, the api
`vim.api.nvim_create_user_command()`
is only available for Neovim 0.7.0+

**Comment by ray-x** [comment]
you can use 0.6 branch
